### ISSUE TITLE
Keyboard and screen reader users currently cannot boing

### ISSUE THREAD

**Issue by zersiax** [gregsadetsky/boing#5]
Interaction appears to be entirely dependent on mouse/touch screen use, might be fun to have a few keyboard shortcuts/a keyboard-only way to trigger boings of various intensities so we don't exclude a bunch of users :)

**Comment by gregsadetsky** [comment]
hey @zersiax thanks for the feedback! I implemented keyboard-driven controls now

try loading/reloading the [page](https://boing.greg.technology/), tabbing once to get to the canvas, pressing spacebar and stretching the spring with the arrow keys. then, release the spacebar to boing. (I realize that there should be better audio feedback when stretching the spring)

let me know what you think! cheers
